README.md

```diff
@@ -968,6 +967,19 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+* <a name="no-nested-modifiers"></a>
+  Avoid nested modifier `if/unless/while/until` usage. Favor `&&/||` if
+  appropriate.
+<sup>[[link](#no-nested-modifiers)]</sup>
```

Comment:
we don't specify this in our version.   I have never seen any dev at this company every do this, so I don't think it's ever come up.